### DISCUSSION TITLE
MapFrom with null coalescing operator

### DISCUSSION THREAD

**Discussion by neozhu** [LuckyPennySoftware/AutoMapper#4004] [NA]
like as 
profile.CreateMap<T,Dto>().ForMember(x => x.EmployeeName, opt => opt.MapFrom(x => x.**Employee?**.Name))

**Comment by lbargaoanu** [comment]
Check [the execution plan](https://docs.automapper.org/en/latest/Understanding-your-mapping.html). That null check is added by default by AM.
